narwhals/_dask/expr.py

```diff
@@ -244,6 +244,12 @@ def sum(self) -> Self:
             "sum",
         )
 
+    def round(self, decimal: int = 0) -> Self:
+        return self._from_call(
+            lambda _input: _input.round(decimal),
+            "round",
+        )
```

Comment:
As we need to pass args and kwargs to `_from_call`, there are a couple of nitty gretty details to take care of.
You can check how this is accomplished in `is_between` a few lines above

Reply:
Looking at `is_between` I altered the code to:
```
def round(self, decimals: int = 0) -> Self:
    return self._from_call(
        lambda _input, decimals: _input.round(decimals),
        "round",
        decimals
    )
```
is this more accurate?

Reply:
Yep that looks on point! 🚀👌

Reply:
@FBruzzesi I've looked into it, but I am not clear on what exactly the additional parameter is supposed to do. Could you please clarify it for me if you can?